docs/source/introduction.rst

```diff
@@ -66,84 +185,126 @@ Additional transaction families provide models for specific areas\:
     * BlockInfo - Provides a methodology for storing information
       about a configurable number of historic blocks.
 
-We also provide an example transaction family to demonstrate
-Sawtooth functionality\:
-
-    * XO - Allows users to play Tic-Tac-Toe, also known as
-      "noughts and crosses" or "Xs and Os".
-
-Consensus
-=========
-
-Consensus is the process of building agreement among a group of mutually
-distrusting participants. There are many different algorithms for building
-consensus based on requirements related to performance, scalability,
-consistency, threat model, and failure model. While most distributed ledgers
-operate with an assumption of Byzantine failures (malicious attacker),
-other properties are largely determined by application requirements.
-For example, ledgers used to record financial transactions often require
-high transaction rates with relatively few participants and immediate
-finality of commitment. Consumer markets, in contrast, require substantial
-aggregate throughput across a large number of participants; however,
-short-term finality is less important.
-
-Algorithms for achieving consensus with arbitrary faults generally require
-some form of voting among a known set of participants. Two general approaches
-have been proposed. The first, often referred to as "Nakamoto consensus",
-elects a leader through some form of “lottery”. The leader then proposes a
-block that can be added to a chain of previously committed blocks. In Bitcoin,
-the first participant to successfully solve a cryptographic puzzle wins
-the leader-election lottery. The elected leader broadcasts the new block
-to the rest of the participants who implicitly vote to accept the block by
-adding the block to a chain of accepted blocks and proposing subsequent
-transaction blocks that build on that chain.
-
-The second approach is based on traditional
-`Byzantine Fault Tolerance (BFT)
-<https://en.wikipedia.org/wiki/Byzantine_fault_tolerance>`_
-algorithms and uses multiple rounds of explicit votes to achieve consensus.
-`Ripple <https://ripple.com/>`_ and `Stellar <https://www.stellar.org/>`_
-developed consensus protocols that extend traditional BFT for open
-participation.
+For more information, see :doc:`transaction_family_specifications`.
+
+
+Real-world Application Examples
+===============================
+
+  * XO: Demonstrates how to construct basic transactions by playing
+    `Tic-tac-toe <https://en.wikipedia.org/wiki/Tic-tac-toe>`_. The
+    XO transaction family includes create and take transactions, with an XO
+    command-line interface (CLI) that allows two participants to play the game.
+    For more information, see
+    :doc:`/app_developers_guide/intro_xo_transaction_family`.
+
+  * Sawtooth Supply Chain:
+    Distributed application (DApp) to help you trace the provenance and other
```

Comment:
[ SHAWN ] Don't mention DApps in this description.  Start with "Helps you trace..."